docs/cloud-tutorials/cicd.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 7
+title: CI/CD tools
+---
+
+This this section we will cover tools provided by DBOS Cloud to enable CI/CD best practices.
+
+## Environments
+Use a different DBOS Cloud application for each environment, _e.g._, dev, staging and production.
+You can do so by deploying the same application multiple times, each time with a different name, _e.g._, `my-app-dev`, `my-app-staging`, `my-app-prod`.
```

Comment:
How do you do this?  And will this even work, given that all three apps will share an `app_db_name` unless we recommend otherwise?